### PR TITLE
Tradeoff Chems

### PR DESCRIPTION
**Pull request by insertnamehere123**
# About the pull request

Adds a trio of chems that heals and restores blood more efficiently at the cost of doing damage:
Alinox heals burns much quicker at the cost of doing oxygen damage.
Yarinox heals brute at the cost of doing toxin damage.
Omniblood restores blood quickly but causes brute damage.

# Explain why it's good for the game

Gives medbay and medics more stuff to play around with.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

786af562ec37d49c3a6ec54ceb60a7d0

</details>

# Changelog

:cl:
add: Added Omniblood, Yarinox and Alinox along with their chem recipes 
balance: New chems means stuff needs to be rebalanced
/:cl:

### CODE REVIEW

**Comment by kiVts** [comment]
wouldnt you be better off creating a new property for each chem? Because we already have transformative which does in summary the same

**Comment by insertnamehere123** [comment]
> 

Probably, but I wanted to make a draft and get feedback on this to maybe improve it

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Downsides are going to be a good bit more.

If it's too easy to offset then it's just literally better and a buff and that's what it is right now.

**Comment by CABSHEP** [comment]
Might want to add conflict with common medicines (like paracetamol and tramadol) that combat the downsides. What is the "tradeoff" of a Omniblood when you pop a bicardine pill on top easly? It would be just a straight marine buff.

**Comment by insertnamehere123** [comment]
> Might want to add conflict with common medicines (like paracetamol and tramadol) that combat the downsides. What is the "tradeoff" of a Omniblood when you pop a bicardine pill on top easly? It would be just a straight marine buff.



> Downsides are going to be a good bit more.
> 
> If it's too easy to offset then it's just literally better and a buff and that's what it is right now.

Actually, that gives me an idea, what if I made it conflict with common meds to make chemicals that cause organ damage or clone damage?

**Comment by insertnamehere123** [comment]
That gives me another thought, what if the conflicts were made a little less obvious? Like Alinox could mix with Bica to make a chem that causes rapid bloodloss? or Yarinox to mix with tramadol or oxy to make a chem that makes causes more eye damage than Imialzadone (the first chem in IA) can heal?

**Comment by insertnamehere123** [comment]
that way one mistake can cause a cascade of failures that guarantees the patient would need to return shipside

File: code/modules/reagents/chemistry_reagents/medical.dm [hunk 1]
```diff
@@ -280,6 +302,15 @@
...
 	properties = list(PROPERTY_UNKNOWN = 6) //handled by blood code
 
+/datum/reagent/medical/omniblood
+	name = "Omniblood"
+	id = "omniblood"
+	description = "A stimulant for the bone marrow that forces it to produce blood cells at an accelerated rate at the cost of degrading bodily tissue."
+	reagent_state = LIQUID
+	color = "#5a0070"
+	chemclass = CHEM_CLASS_UNCOMMON
+	properties = list(PROPERTY_HEMOGENIC = 6, PROPERTY_BIOCIDIC = 1)
```
[the remark this answers is not in the document]
Comment (insertnamehere123) [reply code/modules/reagents/chemistry_reagents/medical.dm:312]: True, but that would make for an interesting drawback to punish overuse of the chem

**Comment by insertnamehere123** [comment]
The more I think about it, the more it just seems like scope creep so I guess I'll just close it until I come up with something better

**Pull request closed without merging**